MeNext/Layout/MainPage.cs

```diff
@@ -63,7 +65,7 @@ public void SomethingChanged()
                     foreach (var page in this.pages) {
                         this.Children.Add(page);
                     }
-                } else {
+                } else { // No Event no tabs.
```

Comment:
Yeah I wasn't too sure about this one, especially since the else was on the same line as the } everywhere else.  

} else {
// Comment about what else means
Else statement

or
//Comment about what else means
} else {
Else statement?